client/components/LearnerViewDirective.js

```diff
@@ -105,25 +133,49 @@ tie.directive('learnerView', [function() {
         var language = LANGUAGE_PYTHON;
         var question = QuestionDataService.getData();
         var prompts = question.getPrompts();
+        var currentPromptIndex = 0;
 
-        var currentPrompt = prompts[0];
-        $scope.title = question.getTitle();
-        $scope.code = question.getStarterCode(language);
-        $scope.instructions = currentPrompt.getInstructions();
-        $scope.feedback = '';
+        $scope.questionIsCompleted = false;
```

Comment:
So here you're using questionIsCompleted, and elsewhere you use isAnswerCorrect() and isLastPrompt(). Is this just a stylistic difference to differentiate between boolean variables and methods that return booleans?